### PR TITLE
Remove composer.phar and add short installation instruction

### PR DESCRIPTION
**Pull request by bocharsky-bw**
Close frozen PR https://github.com/domnikl/DesignPatternsPHP/pull/127 from @TomasVotruba

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,15 @@ This is a collection of known design patterns and some sample code how to implem
 
 I think the problem with patterns is that often people do know them but don't know when to apply which.
 
+## Installation
+To use examples from this repository you should to install dependencies with `Composer` first:
```
Comment (domnikl) [inline README.md]: There is a typo here: "you should ~~to~~ install"

**Comment by TomasVotruba** [comment]
@bocharsky-bw :+1:

**Comment by bocharsky-bw** [comment]
@domnikl Thanks, I'm a bit rephrased installation section and fixed typo.

**Comment by domnikl** [comment]
That's better, thanks! So I am merging it.

**Pull request merged**